src/estimators/any_sketch.py

```diff
@@ -262,7 +309,11 @@ def get_indexes(self, x):
     for index_hashes in self.hash_functions:
       combined_index = []
       for idx_spec, hash_func in zip(self.config.index_specs, index_hashes):
-        combined_index.append(idx_spec.distribution.get_index(hash_func(x)))
+        idx = idx_spec.distribution.get_index(hash_func(x))
+        combined_index.append(idx)
+
+      if len(combined_index) == 0:
+        continue
```

Comment:
I think this if statement can be deleted.

Reply:
Sorry, I tried to reset the branch and probably forgot to save on this file

Reply:
I like to see just:

combined_index.append(idx_spec.distribution.get_index(hash_func(x)))

in the inner loop and everything else removed. We should have the distribution classes ensure that idx < len(idx_spec.distribution). I don't see any reason to allow them to return invalid values.

Reply:
Yeah, it has been already reverted in commit 0001753c805da84a757d99ff24f2490d7795dccc
If you compare the changes since the pull request, it is untouched here